.github/workflows/ci_test_package.yml

```diff
@@ -193,3 +193,86 @@ jobs:
   #       env:
   #         DBT_VERSION: ''
   #       run: tox -e integration_databricks
+
+  integration-sqlserver:
+    strategy:
+      fail-fast: false # Don't fail one DWH if the others fail
+    runs-on: ubuntu-latest
+    environment:
+      name: Approve Integration Tests
+
+    steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.8.x"
+        architecture: "x64"
+    - name: Install SQL Server
+      uses: Particular/install-sql-server-action@v1.2.0
+      with:
+        connection-string-env-var: SQL_SERVER_CONNECTION_STRING
+        catalog: dbt_artifact_integrationtests
+    - name: Create DBT User
+      shell: pwsh
+      run: |
+          echo "Create dbt login with sysadmin"
+          sqlcmd -Q "CREATE LOGIN dbt WITH PASSWORD = '123', CHECK_POLICY = OFF, CHECK_EXPIRATION = OFF" -d "dbt_artifact_integrationtests"
+          sqlcmd -Q "ALTER SERVER ROLE sysadmin ADD MEMBER dbt" -d "dbt_artifact_integrationtests"
+
+    - name: Install tox
+      run: python3 -m pip install tox
+
+    - name: Install Microsoft ODBC
+      run: sudo ACCEPT_EULA=Y apt-get install msodbcsql18 -y
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.sha }} # Check out the code of the PR
+
+    - name: Run Tests on PR
+      env:
+        DBT_VERSION: ${{ matrix.version }}
+      run: tox -e integration_sqlserver
+
+  integration-sqlserver-single-run:
+    strategy:
+      fail-fast: false # Don't fail one DWH if the others fail
+      matrix:
+        # When supporting a new version, update the list here
+        version: ["1_3_0", "1_4_0", "1_7_0", "1_8_0"]
+    runs-on: ubuntu-latest
+    environment:
+      name: Approve Integration Tests
+
+    steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.8.x"
+        architecture: "x64"
+    - name: Install SQL Server
+      uses: Particular/install-sql-server-action@v1.2.0
+      with:
+        connection-string-env-var: SQL_SERVER_CONNECTION_STRING
+        catalog: dbt_artifact_integrationtests
+    - name: Create DBT User
+      shell: pwsh
+      run: |
+          echo "Create dbt login with sysadmin"
+          sqlcmd -Q "CREATE LOGIN dbt WITH PASSWORD = '123', CHECK_POLICY = OFF, CHECK_EXPIRATION = OFF" -d "dbt_artifact_integrationtests"
+          sqlcmd -Q "ALTER SERVER ROLE sysadmin ADD MEMBER dbt" -d "dbt_artifact_integrationtests"
+
+    - name: Install tox
+      run: python3 -m pip install tox
+
+    - name: Install Microsoft ODBC
+      run: sudo ACCEPT_EULA=Y apt-get install msodbcsql18 -y
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.sha }} # Check out the code of the PR
+
+    - name: Run Tests on PR
+      env:
+        DBT_VERSION: ${{ matrix.version }}
```

Comment:
I don't think adding this will achieve anything with the way I'm installing SQL Server. I could copy the code from the file you linked and make that work. Would you prefer that?

Reply:
Yea let's try a different approach to see if it will more reliably connect to sql server

Reply:
Just a heads up, I already changed the way of setting up SQL Server since my last PR. I'm now using this action: https://github.com/Particular/install-sql-server-action . That action should allow for a very easy setup of SQL Server. It supports SQL Sever 2022, 2019 and 2017. I've already tested it in my own fork and the connection has failed me yet. If you can live with this setup I'd prefer not changing it, but I'll go for your setup if required. Let me know what you think!

Reply:
ah ok thanks! It seems like something in the profiles.yml is blocking, could you test my suggestion? When I test on my end postgres fails